### ISSUE TITLE
Textarea input event not firing from inside a shadow DOM when typing

### ISSUE THREAD

**Issue by jmetterrothan** [cypress-io/cypress#8655]
### Current behavior 
With React Unit Testing, when we spy the `input` event of a textarea inside the shadow DOM of a Web Component, we expect to have our function called when we type using 
```js
cy.get('xxx').shadow().get('textarea').type('test')
```
 But no event is triggered using the native `type` method.

Until recently, we used [cypress-shadow-dom](https://github.com/abramenal/cypress-shadow-dom). So we tried to reproduce the test mentioned above with the custom `shadowType` command and it did work fine. It seems the issue is only related to the newly introduced shadow feature.

![image](https://user-images.githubusercontent.com/15093072/94117599-b53f1000-fe4c-11ea-8445-0bb07a0e7cff.png)
The target element for the mouse events is ok, but not for the keyboard events (which is `null`).

### Desired behavior
We would like to have a native `input` event triggered when we use the method `type` inside a web component (shadow DOM).

### Test code to reproduce
I'm not able to provide an example since the source code is confidential and I do not have time to set up a proper test.

### Versions
Cypress : 5.2.0
Cypress Shadow DOM 1.4.1

**Comment by edimitchel** [comment]
@bahmutov @jennifer-shehane sorry for the missing example but this problem is problematic.
This problem is probably caused by the no-native element type of our web component. Cypress don't find the right event to use for.
Could you add an option to force the event to trigger?
Something like that:
```ts
cy.get('...').shadow().find('textarea')
  .type('example{enter}', {
    event: 'input',
    // or
    forceEvent; 'input',
  });
```

**Comment by baermathias** [comment]
Was this solved with with version 9.1.1? 
https://docs.cypress.io/guides/references/changelog#9-1-1
